Classes/Service/StyleguideConfigurationManager.php

```diff
@@ -116,6 +117,22 @@ public function getResponsiveBreakpoints(): array
         return $this->mergedConfiguration['ResponsiveBreakpoints'] ?? [];
     }
 
+    public function getLanguages(): array
+    {
+        return $this->mergedConfiguration['Languages'] ?? [];
+    }
+
+    public function getLanguage($languageKey): ?array
+    {
+        $languageMatch = array_filter(
+            $this->getLanguages(),
+            function ($language) use ($languageKey) {
+                return $language['identifier'] === $languageKey;
+            }
+        );
+        return reset($languageMatch) ?? null;
```

Comment:
`reset` does not work as expected with `??`. If the selected language is not available the function returns `false` instead of `null`.
```suggestion
        return $languageMatch[0] ?? null;
```
or 
```suggestion
        return count($languageMatch) ? reset($languageMatch) : null;
```
To test: Do not define default language next to another in FluidStyleguilde.yaml configuration.
```
  Languages:
    - identifier: de
      locale: de_DE.UTF-8
      hreflang: de
      direction: ltr
      label: Deutsch
```

Reply:
Both issues should be fixed now:
* used `?:` instead of `??` to handle `false` correctly
* use named keys instead of simple array for language, which allows overwriting